superset-frontend/src/SqlLab/components/SaveQuery.tsx

```diff
@@ -114,7 +115,7 @@ export default function SaveQuery({
   const renderModalBody = () => (
     <FormGroup bsSize="small">
       <Row>
-        <Col md={12}>
+        <Col xs={24}>
```

Comment:
This seems OK, but if there's only one `Col` for each of these rows, I'm wondering if we even need the rows/cols at all. It seems like a good ol' `div` would suffice.